test/allowlist_for_publicAPI.json

```diff
@@ -119,51 +119,6 @@
       "Tuple",
       "Union"
   ],
-  "torch.ao.quantization.fx.fuse": [
```

Comment:
just curious, what's the reason these changes are in this PR?

Reply:
nice cleanup!